docker-compose.dev.yml

```diff
@@ -29,7 +31,8 @@ services:
       - '5005:5005' # http
       - '9223:9223' # debug
     volumes:
-      - ./dtfs-central-api/src:/app/src:ro
+      - ./dtfs-central-api/src:/app/dtfs-central-api/src:ro
```

Comment:
Is there a reason to build up individual containers with path convention `/app/[container]/..`, we follow `/app/..` path convention for building containers.